### PR TITLE
Bug 1964748: Merge upstream tag v1.7.2

### PR DESCRIPTION
**Pull request by jmrodri**
Merge upstream tag v1.7.2

    Operator SDK v1.7.2

    Merge executed via ./UPSTREAM-MERGE.sh v1.7.2 upstream master

    Overwritten conflicts:
    Makefile
    images/ansible-operator/Dockerfile
    images/ansible-operator/base.Dockerfile
    images/custom-scorecard-tests/Dockerfile
    images/helm-operator/Dockerfile
    images/operator-sdk/Dockerfile
    images/scorecard-test/Dockerfile
    internal/cmd/ansible-operator/run/cmd.go
    internal/cmd/helm-operator/run/cmd.go
    testdata/ansible/memcached-operator/Dockerfile
    testdata/ansible/memcached-operator/Makefile
    testdata/ansible/memcached-operator/bundle/tests/scorecard/config.yaml
    testdata/ansible/memcached-operator/config/scorecard/patches/basic.config.yaml
    testdata/ansible/memcached-operator/config/scorecard/patches/olm.config.yaml
    testdata/go/v2/memcached-operator/bundle/tests/scorecard/config.yaml
    testdata/go/v2/memcached-operator/config/scorecard/patches/basic.config.yaml
    testdata/go/v2/memcached-operator/config/scorecard/patches/olm.config.yaml
    testdata/go/v3/memcached-operator/bundle/tests/scorecard/config.yaml
    testdata/go/v3/memcached-operator/config/scorecard/patches/basic.config.yaml
    testdata/go/v3/memcached-operator/config/scorecard/patches/olm.config.yaml
    testdata/helm/memcached-operator/Dockerfile
    testdata/helm/memcached-operator/Makefile
    testdata/helm/memcached-operator/bundle/tests/scorecard/config.yaml
    testdata/helm/memcached-operator/config/scorecard/patches/basic.config.yaml
    testdata/helm/memcached-operator/config/scorecard/patches/olm.config.yaml
    website/content/en/docs/installation/_index.md

### CODE REVIEW

**Comment by emmajiafan** [comment]
Finish one regression test, it's ok except the known issue.
run:20210524-0217

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
```
Comment (estroz) [inline Makefile:5]: @jmrodri is `K8S_VERSION` going to get updated to 1.20.2 in a follow-up? Or does that get updated in some other way downstream? It's purely cosmetic downstream, for those curious.

**Pull request merged**